fury/interactor.py

```diff
@@ -142,6 +142,8 @@ def _process_event(self, obj, evt):
             self.on_mouse_wheel_forward(obj, evt)
         elif evt == "MouseWheelBackwardEvent":
             self.on_mouse_wheel_backward(obj, evt)
+        elif evt == "LeftButtonDoubleClickEvent":
```

Comment:
this event does not exist in vtk. We need to create this custom event. I recommend you to comment this part for the moment